make/clean.mk

```diff
@@ -4,4 +4,6 @@ clean:
 	@cd "$(GOPATH)/src/github.com/codeready-toolchain/registration-service" && \
 		rm -f pkg/static/generated_assets.go && \
 		rm -rf $(COV_DIR) && \
-		rm -f registration-service
+		rm -f registration-service && \
```

Comment:
shouldn't it be `./bin/registration-service` instead? The Makefile `build-dev` and `build-prod` goals generate the binary in `-o $(OUT_DIR)/bin/registration-service`

Reply:
Right. It’s a leftover from the previous binary path.

Reply:
Done in 8af6a1af6a8d69f5871dffae62436a7dd8b011f2